ui/app/rspack.config.mjs

```diff
@@ -29,6 +28,13 @@ export default defineConfig({
   entry: './src/bundle.ts',
   resolve: {
     extensions: ['...', '.ts', '.tsx', '.jsx'],
+    alias: {
+      '@perses-dev/core': resolve(import.meta.dirname, '../core/dist/src'),
+      '@perses-dev/components': resolve(import.meta.dirname, '../components/dist/src'),
+      '@perses-dev/dashboards': resolve(import.meta.dirname, '../dashboards/dist/src'),
+      '@perses-dev/explore': resolve(import.meta.dirname, '../explore/dist/src'),
+      '@perses-dev/plugin-system': resolve(import.meta.dirname, '../plugin-system/dist/src'),
```

Comment:
that's interesting, curious to know why this is needed now.

Reply:
Now the plugin needs a little bit of help to locate the monorepo packages. I adjusted it to use the ts paths so is more future proof